setup.py

```diff
@@ -58,7 +59,8 @@ def load_deps(file_name):
     author_email="yaronh@iguazio.com",
     license="Apache",
     url="https://github.com/mlrun/storey",
-    packages=find_packages(),
+    packages=find_packages(include=["storey*"]),
+    python_requires=">=3.9",
```

Comment:
I guess we might as well because mlrun dropped older versions and 3.8 EOL is very near anyway.

Reply:
I didn't understand your comment here - we might as well what?